test/message_test.rb

```diff
@@ -1,6 +1,11 @@
 require 'test_helper'
 
 class MessageTest < Minitest::Test
+
+  def positionmessage
```

Comment:
It makes sense to leave the position message. as a variable to use in multiple places. I would prefer to declare it as an instance variable in a setup block however, as opposed to a helper method near the top of the file. To stay in line with naming conventions, I would also rename the variable to `position_message`.